doc/reported_states_during_deployment.md

```diff
@@ -0,0 +1,353 @@
+
+# Tracking $edgeAgent's reported properties during deployment.
+
+An IoT Edge runtime receives a new deployment through \$edgeAgent's twin in the desired properties 
+section.  When it receives this new deployment, it creates a plan to apply changes to the current 
+system. Once that plan finishes executing, successfully or not, $edgeAgent will update its 
+reported properties.  
+
+This document will help spell out some of the details of \$edgeAgent's twin properties as the IoT 
+Edge runtime applies a new deployment. 
+
+# Relevant twin properties
+
+There are two sections describing relevant twin properties below. The first describes properties 
+set on the Iot Hub service and read by the IoT Edge. The second describes properties set by the IoT Edge
+and reported back to the service.
+
+## Set by IoT Hub service
+
+The following sections are set in \$edgeAgent twin by the IoT Hub service and are read by the IoT Edge 
+runtime:
+
+### Configuration section
+
+This section is present when the service applies at-scale (base and layered) deployments. This 
+section will not be present if no at-scale or layered deployment is applied.
+
+Example:
+
+```json
+{
+    "configurations" : {
+        "at-scale1": {
+            "status": "Applied"
+        },
+        "at-scale2": {
+            "status": "Targeted"
+        }
+    }
+}
+```
+
+| Relevant Field | Meaning |
+|------------------------------------------|-------------------------------------------------------|
+| .configurations.*\<deployment name\>*.status | "Applied" or "Targeted" for each deployment that may be targeted to this device. |
+
+
+### Desired Properties section
+
+This is the \$edgeAgent section of the modulesContent in an edge deployment.
```

Comment:
please add a special styling. Either _modulesContent_ or **modulesContent** to emphasis it is the actual field name.